src/Cms/App.php

```diff
@@ -848,6 +849,7 @@ protected function optionsFromProps(array $options = []): array
 
     /**
      * Merge last-minute options from ready callback
+     * @since 3.3.0
```

Comment:
Do we need that one as that's a protected method?

Reply:
I wondered as well, but then thought that this type if information is generally useful

Reply:
I'm not sure. Protected methods are internal implementation details, so it shouldn't matter when they were changed/added. It would be a lot of work to add that information everywhere, so I think that it isn't worth it for the limited use.

👍 for all other changes though!